templates/html.tpl.php

```diff
@@ -44,7 +44,7 @@
 <body class="<?php print $classes; ?>" <?php print $attributes;?>>
   <?php if ($skip_link_text && $skip_link_anchor): ?>
     <p id="skip-link">
-      <a href="#main-content" class="element-invisible element-focusable"><?php print $skip_link_text; ?></a>
+      <a href="#" onclick="document.getElementById('base-srch').focus({preventScroll:false});" class="element-invisible element-focusable"><?php print t('Skip to main search.'); ?></a>
```

Comment:
Does this work in IE?

Reply:
@walkerjam, I don't have a PC with IE set up to test this on, but I did check MDN and caniuse.com, and it looks like the plan `.focus()` method works at least as early as IE 9 and that the scroll toggle works in 10/11. I also did a quick browser stack test (they added it to caniuse.com), and it worked in their IE 11.